packages/backend.ai-ui/vite.config.ts

```diff
@@ -9,6 +10,18 @@ const __dirname = dirname(fileURLToPath(import.meta.url));
 
 export default defineConfig(({ mode }) => {
   const isDevMode = mode === 'development';
+
```

Comment:
A little tip: When writing code paragraphs for setting variables, don't leave blank lines between the variable declarations. Separating with blank lines creates distinct paragraphs, which should represent grouped code blocks that serve the same purpose.